utils/constants.js

```diff
@@ -1,4 +1,4 @@
-const appTemplateFileExclusions = ['package.json'];
+const appTemplateFileExclusions = [];
```

Comment:
How is this going to affect the generator??

Reply:
in index.js  we have appTemplateFileExclusions which exclude the package.json
```
const copyBaseDirectory = () => {
  copyDir(baseTemplatePath, projectDir, appTemplateFileExclusions);
};
```